responders/src/UI/embc-responder/src/app/app.component.spec.ts

```diff
@@ -128,7 +149,7 @@ describe('AppComponent', () => {
     };
     fixture.detectChanges();
     component.ngOnInit();
-    expect(component.environment.envName).toContain(
+    expect(component.envBannerService.environmentBanner.envName).toContain(
```

Comment:
Incorrect test case, you are comparing service with service. Compare the component variable with the service